.claude/hooks/utils/llm/oai.py

```
@@ -0,0 +1,172 @@
+    
+    # Try loading from each possible location
+    env_loaded = False
+    for env_path in possible_env_paths:
+        if os.path.exists(env_path):
+            load_dotenv(env_path)
+            env_loaded = True
+            # Log to same file as summarizer for debugging
+            try:
+                with open(".claude/logs/summarizer.log", "a") as f:
+                    f.write(f"[DEBUG] Loaded .env from: {env_path}\n")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)